src/pages/_document.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
```

Comment:
Correct me if I'm wrong, but since we're using React 17, I don't believe it's necessary to `import React from 'react'` anymore, here or elsewhere.

Reply:
Needed to update an eslint rule. Should be good now.